dev-docs/source/GettingStartedWithPyCharm.rst

```diff
@@ -46,8 +48,35 @@ PyCharm should be opened using ``pycharm.bat`` which can be found in the build d
 
 NOTE : In some cases, imports in the code will still be highlighted red when they come from folders within the ``script/`` folder, or from other folders entirely. To fix this simply add the relevant folder that contains the module you are importing in the same fashion as step 3 above.
 
-Running Files in the Debugger
------------------------------
+Running Files in the Debugger with EnvFile extension
+----------------------------------------------------
+
+4. Install the EnvFile plugin by Borys Pierov. The plugin can be installed in multiple ways:
+
+   a) Open Settings(CTRL + SHIFT + S), to go Plugins and search for ``EnvFile``. Install and restart PyCharm.
+   b) Go to the plugin's `webpage <https://plugins.jetbrains.com/plugin/7861-envfile>`_, download and install it.
+
+5. To edit the configurations go to Run->Run... and select Edit Configurations. Notice that there is now a ``EnvFile`` tab under the configuration's name.
+   - Note that you have to do that for each configuration, or you can change the template configuration, and all configuration that use that template will have the EnvFile setup.
+6. Open the ``EnvFile`` tab, check ``Enable EnvFile`` and ``Substitute Environmental Variables (...)`` - this allows setting up the third-party paths dynamically.
+7. Click the ``+`` (plus) on the right side, select the ``pycharm.env`` file in the root of the **build** directory.
```

Comment:
How does this deal with Release/Debug? Do you still need to add separate run configurations for them?

Reply:
Yes, this will only set up the environment variables, there is no access to the execution script and working directory that need to be set up in the configurations sadly.